products/accesscontextmanager/terraform.yaml

```diff
@@ -50,6 +50,27 @@ overrides: !ruby/object:Provider::ResourceOverrides
       pre_update: templates/terraform/pre_update/update_mask.erb
       encoder: templates/terraform/encoders/access_level_never_send_parent.go.erb
       custom_import: templates/terraform/custom_import/access_level_self_link_as_name_and_set_parent.go.erb
+  ServicePerimeter: !ruby/object:Provider::Terraform::ResourceOverride
+    id_format: "{{name}}"
+    import_format: ["{{name}}"]
+    example:
+      - !ruby/object:Provider::Terraform::Examples
+        name: "access_context_manager_service_perimeter_basic"
+        skip_test: true
```

Comment:
It looks like `skip_test` doesn't generate oics, examples or tests. What does this block provide if skip_tests is true?

Reply:
The old-style examples are going to be removed when I take a few minutes to do it & only these ones will be left.

Reply:
Nope, just that I don't support generating tests that will pass for the resource yet! Ideally no resources would have `skip_test` and every resource would have a generated test file + have OiCS.

This one is an organization-level resource, and the parent AccessPolicy is globally unique and neither of those are problems I think are worth solving yet (vs just converting more compute resources to MM, for example)